src/class/net/net_device.c

```diff
@@ -44,6 +44,7 @@ typedef struct
   uint8_t ep_notif;
   uint8_t ep_in;
   uint8_t ep_out;
+  uint_fast8_t active_alt_setting;
```

Comment:
please use uint8_t, I like the type is exactly fixed width on all platform.

Reply:
Actually, I'll remove all changes to that file before merging. (The changes don't work, anyway)